pkg/controller/node/range_allocator.go

```diff
@@ -239,9 +240,20 @@ func (r *rangeAllocator) updateCIDRAllocation(data nodeAndCIDR) error {
 			}
 			return nil
 		}
+		oldData, err := json.Marshal(node)
```

Comment:
We shouldn't use "json.Marshal" - instead we should be using this:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/json/json.go#L33

Even though for Marshal it's the same, if we decide to Unmarshal something, there will be a temptation to use the same, so using the other package is safer.